snekbox/api/resources/eval.py

```diff
@@ -54,6 +61,29 @@ class EvalResource:
     def __init__(self, nsjail: NsJail):
         self.nsjail = nsjail
 
+    @validate(
+        resp_schema={
+            "versions": {"type": "array", "items": {"type": "str"}},
+        }
+    )
+    def on_get(self, _: falcon.Request, resp: falcon.Response) -> None:
```

Comment:
Using GET on `/eval` to get this info seems unintuitive. I think this is better suited to be a dedicated "info" route. Related: #138

Reply:
Sounds good, will change.

Reply:
Done in 8375787. I didn't implement the rest of the suggestions from that issue, but I'd think they'd be easy to implement in a future PR.